### DISCUSSION TITLE
Questions about disk quota limits for specific databases

### DISCUSSION THREAD

**Discussion by george-umc** [milvus-io/milvus#38029] [Q&A and General discussion] [answered]
Hello,

I have a few questions regarding disk usage limits.

Currently, I am using Docker Compose to deploy the standalone version of Milvus v2.4.11

1. Is it possible to set disk quotas for specific databases in Milvus? ex: a-db 50GB, b-db 100GB
    I found the setting `quotaAndLimits.limitWriting.diskProtection.diskQuotaPerDB`, but it doesn't seem to allow setting quotas for specific databases.

2. Is there a way to check the disk usage for a specific database?

**Comment by yhmo** [comment, marked as answer]
Currently, it cannot limit disk usage for a specific db. The quota is limited in 4 levels: global, db, collection, and partition. We need to add more configurations if we decide to allow disk quota for a specific database, specific collection, or specific partition.

Wait @xiaofan-luan to comment.

**Reply by george-umc** [reply]
Based on what you mentioned, can I conclude that Milvus currently does not support checking the disk quota for a specific database?

**Reply by yhmo** [reply]
Right, the **diskQuotaPerDB** is for each database, not for a specific database.
